eng/targets/ReferenceAssembly.targets

```diff
@@ -55,7 +55,7 @@
     </ItemGroup>
 
     <PropertyGroup>
-      <_GenAPICommand Condition="'$(MSBuildRuntimeType)' == 'core'">$(ToolHostCmd) --roll-forward-on-no-candidate-fx 2 "$(_GenAPIPath)"</_GenAPICommand>
+      <_GenAPICommand Condition="'$(MSBuildRuntimeType)' == 'core'">"$(DotNetTool)" --roll-forward-on-no-candidate-fx 2 "$(_GenAPIPath)"</_GenAPICommand>
```

Comment:
Have we already taken an Arcade dependency update with this change in it? If so, how did validation succeed for that PR?

Reply:
PR validation runs with `'$(MSBuildRuntimeType)' == 'Full'`